docs/dev-guides/PERSONAL-ENVIRONMENTS.md

```diff
@@ -0,0 +1,523 @@
+```
+```text
+CURRENT   NAME                                                   CLUSTER                                                AUTHINFO                                               NAMESPACE
+*         gke_terra-kernel-k8s_us-central1-a_terra-integration   gke_terra-kernel-k8s_us-central1-a_terra-integration   gke_terra-kernel-k8s_us-central1-a_terra-integration   terra-jcarlton
+```
+### Getting Descriptions
+To get the description of the namespace, use `describe namespace`
+```shell script
+$ kubectl describe namespace terra-jcarlton
+```
+```text
+Name:         terra-jcarlton
+Labels:       <none>
+Annotations:  <none>
+Status:       Active
+
+Resource Quotas
+ Name:                       gke-resource-quotas
+ Resource                    Used  Hard
+ --------                    ---   ---
+ count/ingresses.extensions  1     5k
+ count/jobs.batch            0     10k
+ pods                        1     5k
+ services                    1     1500
+
+No LimitRange resource.
+```
+
+The output shows that I have an active namespace with no labels and I'm well under quota. To view details
+about the pods, use `get pods` and isolate the namespace to use:
+```shell script
+$ kubectl get pods --namespace=terra-jcarlton
+```
+```text
+NAME                                           READY   STATUS     RESTARTS   AGE
+workspacemanager-deployment-68dfc86fbb-zhpwc   0/3     Init:0/1   0          4h35m
+```
+ For the service, use `get services`
+ ```shell script
+$ kubectl get services --namespace=terra-jcarlton
+```
+```text
+NAME                       TYPE        CLUSTER-IP    EXTERNAL-IP   PORT(S)   AGE
+workspacemanager-service   ClusterIP   10.56.8.103   <none>        443/TCP   4h39m
+```
+
+It's also possible to set this namespace as a default, at which point you can leave off the `--namespace`
+qualifier:
+```shell script
+$ kubectl config set-context --current --namespace=terra-jcarlton
+```
+```text
+Context "gke_terra-kernel-k8s_us-central1-a_terra-integration" modified.
+```
+
+To list the nodes in the cluster do
+```shell script
+kubectl get nodes
+```
+
+<details>
+
+<summary>Node list</summary>
+
+```text
+NAME                                             STATUS   ROLES    AGE   VERSION
+gke-terra-integration-cronjob-v1-9d4ae6cb-dvz6   Ready    <none>   31h   v1.17.14-gke.400
+gke-terra-integration-default-c1ae3b79-hv62      Ready    <none>   30h   v1.17.14-gke.400
+gke-terra-integration-default-c1ae3b79-jrmn      Ready    <none>   30h   v1.17.14-gke.400
+gke-terra-integration-default-c1ae3b79-m2ui      Ready    <none>   30h   v1.17.14-gke.400
+gke-terra-integration-default-c1ae3b79-pa8n      Ready    <none>   30h   v1.17.14-gke.400
+gke-terra-integration-default-c1ae3b79-qstb      Ready    <none>   30h   v1.17.14-gke.400
+gke-terra-integration-default-c1ae3b79-rert      Ready    <none>   30h   v1.17.14-gke.400
+gke-terra-integration-default-v2-95d71f95-gdfy   Ready    <none>   30h   v1.17.14-gke.400
+```
+
+</details>
```

Comment:
I'd add something about how to get started in swagger ui. In particular, it confused me that for resource buffer I needed to glogin as a service account, while for wsm it had to be my broad account (that may be a bug somewhere)

Reply:
fwiw, they're configured differently somewhat on purpose. buffer is never supposed to be user visible, so user accounts should not have access to its endpoints. WM should be accessible by users, but we don't want our dev instances open to the public. (Last I heard this was going to be blocked by VPN, but I guess that's changed.)

Reply:
I propose we do a separate dev guide for working with Swagger UI. It has more to do with services than with environments from what I gather.